src/charm.py

```diff
@@ -989,6 +1199,11 @@ def _is_relation_created(self, relation_name: str) -> bool:
         """
         return bool(self.model.get_relation(relation_name))
 
+    def _is_tls_certificates_access_relation_exists(self) -> bool:
+        """Check if the tls-certificates-access relation exists."""
+        logger.warning(self.model.get_relation(TLS_CERTIFICATE_ACCESS_RELATION_NAME))
+        return bool(self.model.get_relation(TLS_CERTIFICATE_ACCESS_RELATION_NAME))
```

Comment:
This is not used anywhere

Reply:
I stopped needing this function in one of the reworks I did. Removed.